inputters/pandocast.lua

```diff
@@ -84,7 +84,7 @@ local function pandocAstParse(element)
     if element.t then
       if Pandoc[element.t] then
         if HasMultipleArgs[element.t] then
-          addNodes(out, Pandoc[element.t](table.unpack(element.c)))
+          addNodes(out, Pandoc[element.t](pl.utils.unpack(element.c)))
```

Comment:
This PR raises a conflict somewhere near these lines -- I realize it was made on a former version, probably 1.4.0 from June 2023. Can you possibly rebase your changes on current master (= 1.5.0 at this date).
No urge - I still have to manage having a SILE build based on Lua 5.1.

Reply:
Done in #102

Reply:
Just to confuse the issue, since we're loading `compat53` the `table.pack()` function is actually being patched and/or provided behind the scenes. I've been trying *not* to rely on that where possible and explicitly using `pl.utils.<func>` or other explicit calls where possible in the hopes that someday we don't need a monkey-patched set of Lua internal functions. The side-effects of `compat53` are not nearly as egregious as `std` were (so glad we got rid of that!) but sometimes they are still unexpected, especially when 3rd party dependencies don't expect it.

Reply:
Noted.
Here I suspect (without having checked) that it would _perhaps_ work even if we didn't have compat53, as we load jgm's lunamark or djot library, and they also have their own compatibility shims...

Reply:
Welcome to Lua. It's shims all the way down.